File: 06-06-2025-C#-Final/Freelance-Project/Models/DTO/Summary/ProposalSummary.cs
namespace Freelance_Project.Models.DTO;

public class ProposalSummaryDTO
{
    public Guid Id { get; set; }
    public Guid FreelancerId { get; set; }
    public decimal ProposedAmount { get; set; }
    public TimeSpan? ProposedDuration { get; set; }
    public bool IsActive { get; set; }
}

File: 23-06-2025-Angular-Final/Freelance-Project/src/app/Models/Summary/ProposalSummary.ts
export class ProposalSummary {
    public id: string;
    public freelancerId: string;
    public proposedAmount: number;
    public proposedDuration: string | null;
    public isActive: boolean;

    constructor() {
        this.id = '';
        this.freelancerId = '';
        this.proposedAmount = 0;
        this.proposedDuration = null;
        this.isActive = true;
    }
}
